### PR TITLE
Make logistration pages responsive.

### PR DESCRIPTION
**Pull request by waheedahmed**
Made logistration pages response for smaller devices using paragon components and bootstrap classes.

VAN-222

**Sandbox:** https://logistration-van-222.sandbox.edx.org/login

### CODE REVIEW

File: src/logistration/InstitutionLogistration.jsx [hunk 1]
```diff
@@ -34,23 +34,22 @@ const InstitutionLogistration = props => {
...
+          <h3 className="mt-5 mb-4 font-weight-normal">
             {headingTitle}
-          </p>
-          <div style={{ fontSize: '16px' }}>
-            <p
-              className="mb-2"
-              style={{ fontSize: '16px' }}
-            >
+          </h3>
+          <div className="">
```
Comment (zainab-amir) [inline src/logistration/InstitutionLogistration.jsx:42]: nit: remove empty className

File: src/logistration/_style.scss [hunk 2]
```diff
@@ -21,8 +21,6 @@ $apple-focus-black: $apple-black;
 
 .register-container {
   @extend .login-container;
-
-  width: 30rem;
```
Comment (zainab-amir) [inline src/logistration/_style.scss:25]: if login-container is not used anywhere else can we remove it and just add the values directly to register-container?

**Review by zainab-amir — APPROVED** [review]
Looks good 💯 Just a couple of nits

File: src/logistration/_style.scss (hunk 2 above)
Comment (waheedahmed) [reply src/logistration/_style.scss:25]: @zainab-amir Actually both are not being used, I'm gonna remove both.

**Pull request merged**